Javascript/variables.js

```diff
@@ -161,35 +163,35 @@ var ordinals = ["first", "second", "third", "fourth", "fifth", "sixth", "seventh
 var cardinals = ["one", "two", "three", "four", "five", "six", "seven", "eight", "nine", "ten", "eleven", "twelve", "thirteen", "fourteen", "fifteen", "sixteen", "seventeen", "eighteen", "nineteen", "twenty", "twentyone", "twentytwo", "twentythree", "twentyfour", "twentyfive", "twentysix", "twentyseven", "twentyeight", "twentynine", "thirty", "thirtyone", "thirtytwo", "thirtythree", "thirtyfour"]
 
 var challengebaserequirements = {

```

Comment:
`let` instead

Reply:
let stops the variable from being global, and in this case it is global, so it has to be var.